### PR TITLE
Fix ArchivedTabView and tabgrid overlap for landscape mode

### PR DESCRIPTION
**Pull request by yasminyin96**
* Fix ArchivedTabView and EmptyCardGrid overlap for landscape mode and iPad

### How was this tested?
- [X] I tested this manually

### Screenshots and screen recordings

https://user-images.githubusercontent.com/104786010/172730122-dd5a7ebf-02b0-4dcf-bb7c-aca45ebf41a6.mov

https://user-images.githubusercontent.com/104786010/172730140-02566885-e885-4253-8adf-4e057b222956.mov

https://user-images.githubusercontent.com/104786010/172730148-c8a2c967-820b-41aa-a231-f0303148fb32.mov

https://user-images.githubusercontent.com/104786010/172730153-64066ba4-f535-428a-84c4-5f5101497d51.mov

## Issues addressed
* https://github.com/neevaco/neeva-ios/issues/3790

### CODE REVIEW

File: Client/Frontend/Browser/Card/Grid/CardsContainer.swift [hunk 1]
```diff
@@ -41,6 +41,7 @@ struct TabGridContainer: View {
 
     @Environment(\.columns) private var columns
     @State var cardStackGeom: CGSize = CGSize.zero
+    @State var oniPad = UIDevice.current.userInterfaceIdiom == .pad
```
[the remark this answers is not in the document]
Comment (yasminyin96) [reply Client/Frontend/Browser/Card/Grid/CardsContainer.swift:44]: Good point.

File: Client/Frontend/Browser/Card/Grid/EmptyCardGrid.swift [hunk 2]
```diff
@@ -2,19 +2,29 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import Shared
 import SwiftUI
 
 struct EmptyCardGrid: View {
     let isIncognito: Bool
     let isTopBar: Bool
+    let isCardEmtpy: Bool
```
[the remark this answers is not in the document]
Comment (yasminyin96) [reply Client/Frontend/Browser/Card/Grid/EmptyCardGrid.swift:11]: For **isCardEmpty**, I just pass the value of **tabModel.normalDetails.isEmpty** to the EmptyCardGird.

What do you think that rename it to **showArchivedTabsView** and then pass the value of 
 **FeatureFlag[.enableArchivedTabsView] && !isIncognito && tabModel.normalDetails.isEmpty && !oniPad** .

[the remark this answers is not in the document]
Comment (evancrow) [reply Client/Frontend/Browser/Card/Grid/EmptyCardGrid.swift:11]: Sounds good to me

**Review by evancrow — APPROVED** [review]
LGTM - looks like there's a test failure

**Comment by chassu** [comment]
Yasmin says there seems to be some issues with iPad Mini. Will wait on this before I merge #3812

**Pull request merged**